modules/cert-manager-istio-csr-config-ca-cert.adoc

```diff
@@ -0,0 +1,47 @@
+:_mod-docs-content-type: PROCEDURE
+[id="cert-manager-istio-csr-config-ca-cert_{context}"]
+= Configuring the CA certificate for the Istio server
+
+You can configure the `ConfigMap` that contains the CA bundle used by Istio workloads to verify the Istio server certificate. If not configured, the cert-manager Operator looks for the CA certificate in the configured issuer and in the Kubernetes Secret that contains the Istio certificates.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have created the `IstioCSR` custom resource (CR).
+
+.Procedure
+
+. Edit the `IstioCSR` CR by running the following command:
++
+[source,terminal]
+----
+oc edit istiocsrs.operator.openshift.io default -n <istiocsr_namespace> <1>
+----
+<1> Replace `<istiocsr_namespace>` with the namespace where you created the `IstioCSR` CR.
+
+. Configure the CA bundle by editing the `spec.istioCSRConfig.certManager` section:
++
+.Sample `IstioCSR` CR with CA bundle configuration
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1alpha1
+kind: IstioCSR
+...
+spec:
+  istioCSRConfig:
+    certManager:
+      istioCACertificate:
+        key: <key_in_the_configmap> <1>
+        name: <configmap_name> <2>
+        namespace: <configmap_namespace> <3>
+----
+<1> Specify the key name in the `ConfigMap` that contains the CA bundle.  
+<2> Specify the name of the `ConfigMap`.  
```

Comment:
Should we mention somewhere that the Configmap and the key must exist? prior to updating these entries.